todo_app/app.py

```diff
@@ -1,43 +1,45 @@
+from todo_app.mongo.mongo_wrapper import MongoWrapper
 from flask import Flask, render_template, request, redirect, url_for
 import os
-from todo_app.trello.trello_api import TrelloAPI
 from todo_app.models.view_model import ViewModel
 
 
 def create_app():
     app = Flask(__name__)
-    trello_board = TrelloAPI(os.environ['TRELLO_BOARD_ID'])
+    mongo_database = MongoWrapper(os.environ['DEFAULT_DATABASE'])
```

Comment:
Not important, but worth considering - we're making our app very aware of the database by using variable names like this, and setting it up as this wrapper. You've been effective at keeping the external interface the same so that you didn't need to change arguments passed in etc so even without trying to hide it properly, choosing a more generic name like `task_repository` or something would potentially mean this is the only line that would need to change in this file if we were to switch to another storage solution